dts/bindings/mtd/nxp,imx-xspi-mx25um51345g.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP XSPI MX25UM51345G
+
+compatible: "nxp,imx-xspi-mx25um51345g"
```

Comment:
Let's avoid using `imx-` in the naming scheme, just use `nxp,xpi`

Reply:
Updated.